runtime/primitives/src/xcm.rs

```diff
@@ -0,0 +1,243 @@
+// Copyright 2020-2022 Manta Network.
```

Comment:
Can we move these xcm related code to a separated crate like `xcm-support` or `?

Reply:
Let's leave this to another PR.